### PR TITLE
added loadBalancerIP option to kibana service

### PR DESCRIPTION
**Pull request by pixxet**
- [ √] Chart version *not* bumped (the versions are all bumped and released at the same time)
- [ √] README.md updated with any new values or changes
- [ √] Updated template tests in `${CHART}/tests/*.py` 
- [ √] Updated integration tests in `${CHART}/examples/*/test/goss.yaml`

### CODE REVIEW

**Comment by pixxet** [comment]
CLA signed

File: kibana/tests/kibana_test.py [hunk 1]
```diff
@@ -427,6 +427,21 @@ def test_service_annotations():
     ]
     assert s == "0.0.0.0/0"
 
+def test_adding_a_loadBalancerIP():
```
Comment (jmlrt) [inline kibana/tests/kibana_test.py:430]: Black formatter requires 2 blanck lines between each functions:
```suggestion

def test_adding_a_loadBalancerIP():
```

File: kibana/tests/kibana_test.py [hunk 2]
```diff
@@ -427,6 +427,21 @@ def test_service_annotations():
...
+    assert "loadBalancerIP" not in r["service"][uname]["spec"]
+
+    config = """
+    service:
+      loadBalancerIP: 12.4.19.82
+    """
+
+    r = helm_template(config)
+
+    assert r["service"][uname]["spec"]["loadBalancerIP"] == "12.4.19.82"
```
Comment (jmlrt) [inline kibana/tests/kibana_test.py:444]: Black formatter requires 2 blanck lines between each functions:
```suggestion
    assert r["service"][uname]["spec"]["loadBalancerIP"] == "12.4.19.82"
    
```

**Comment by jmlrt** [comment]
jenkins test this please

**Comment by jmlrt** [comment]
cla/check

File: kibana/tests/kibana_test.py [hunk 3]
```diff
@@ -428,6 +428,23 @@ def test_service_annotations():
     assert s == "0.0.0.0/0"
 
 
+def test_adding_a_loadBalancerIP():
+    config = ""
+
+    r = helm_template(config)
+
+    assert "loadBalancerIP" not in r["service"][uname]["spec"]
```
Comment (jmlrt) [inline kibana/tests/kibana_test.py:436]:
```suggestion
    assert "loadBalancerIP" not in r["service"][name]["spec"]
```

**Comment by jmlrt** [comment]
CLA is still failling, can you ensure that your commits are matching the email you used to sign CLA?

**Pull request closed without merging**